From: Michael Tokarev <mjt@tls.msk.ru>
Date: Fri, 01 Mar 2024 16:16:20 +0300
Subject: lower dns lookup mismatch messages

There's just too much useless noise in the logs,
especially when the PTR record just doesn't exist.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>

diff --git a/source3/lib/util_sock.c b/source3/lib/util_sock.c
index 5ff875c230d..b9237c3797e 100644
--- a/source3/lib/util_sock.c
+++ b/source3/lib/util_sock.c
@@ -710,5 +710,5 @@ static bool matchname(const char *remotehost,
 		(!strequal(remotehost, ailist->ai_canonname) &&
 		 !strequal(remotehost, "localhost"))) {
-		DEBUG(0,("matchname: host name/name mismatch: %s != %s\n",
+		DEBUG(5,("matchname: host name/name mismatch: %s != %s\n",
 			 remotehost,
 			 ailist->ai_canonname ?
@@ -736,5 +736,5 @@ static bool matchname(const char *remotehost,
 	 */
 
-	DEBUG(0,("matchname: host name/address mismatch: %s != %s\n",
+	DEBUG(5,("matchname: host name/address mismatch: %s != %s\n",
 		print_sockaddr_len(addr_buf,
 			sizeof(addr_buf),
@@ -892,5 +892,5 @@ int get_remote_hostname(const struct tsocket_address *remote_address,
 	} else {
 		if (!matchname(name_buf, (struct sockaddr *)&ss, len)) {
-			DEBUG(0,("matchname failed on %s\n", name_buf));
+			DEBUG(5,("matchname failed on %s\n", name_buf));
 			strlcpy(name_buf, "UNKNOWN", sizeof(name_buf));
 		}
